stm32-data-gen/src/chips.rs

```diff
@@ -514,6 +514,7 @@ impl PeriMatcher {
             ("STM32L4.*:GFXMMU:.*", ("gfxmmu", "v1", "GFXMMU")),
             ("STM32U5.*:GFXMMU:.*", ("gfxmmu", "v2", "GFXMMU")),
             ("STM32U5.*:ICACHE:.*", ("icache", "v1", "ICACHE")),
+            ("STM32U5.*:CORDIC:.*", ("cordic", "v1", "CORDIC")),
```

Comment:
```suggestion
            (".*:CORDIC:.*", ("cordic", "v1", "CORDIC")),
```

All STM chips with CORDIC support (G4, H7, H5, U5) have the same registers, so let's just expose it for all of them.